common/config/azure-pipelines/jobs/regression-testing.yaml

```diff
@@ -28,6 +28,21 @@ jobs:
 
     strategy:
       matrix:
+        "Windows_Node_20_x":
+          OS: windows-latest
+          platform: Windows_NT
+          name: $(win_pool)
+          node_version: 20.x
```

Comment:
should this be 20.x or the first lts version?

Reply:
Comment from the file:
> Starts with the minimum Node version (currently 18.12.0) and then follows the tip of each subsequent major LTS (even-numbered) version.

I assume, this means we need to test latest version of Node 20 LTS